Makefile

```
@@ -0,0 +1,1554 @@
+# Author: y_ohi
+# Description: Comprehensive Ubuntu setup with applications and dotfiles configuration
+
+.PHONY: all help system-setup install-homebrew install-apps install-deb-packages install-flatpak-packages \
+        setup-vim setup-zsh setup-wezterm setup-vscode setup-cursor setup-git setup-docker setup-development setup-shortcuts \
+        setup-gnome-extensions setup-gnome-tweaks backup-gnome-tweaks export-gnome-tweaks setup-all clean system-config clean-repos install-cursor install-fuse install-cica-fonts install-ibm-plex-fonts install-mysql-workbench
+
+# デフォルトターゲット
+all: help
+
+# ヘルプメッセージ
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (61).

(maxbodylength)